packages/bcer-api/app/src/location/entities/location.entity.ts

```diff
@@ -97,6 +98,16 @@ export class LocationEntity {
   @JoinTable()
   manufactures: ManufacturingEntity[];
 
+  @Column('enum', {
+    enum: LocationStatus,
+    nullable: false,
+    default: LocationStatus.ACTIVE,
+  })
+  status: LocationStatus
+
+  @Column({ type: 'timestamp', nullable: true, default: null })
+  closed_at: Date;
```

Comment:
in js, we use `closedAt` and put the database column name in `@Column({..., name: 'closed_at'})`